addons/web/static/src/views/pivot/pivot_renderer.xml

```diff
@@ -17,7 +17,7 @@
                                 <t t-set="isXAxis" t-value="true"/>
                                 <t t-set="isInHead" t-value="true"/>
                             </t>
-                            <th t-else="" t-att-colspan="cell.width" t-att-rowspan="cell.height" class="fw-normal border-start-0" t-att-class="{ 'border-top': cell_index != 0 }"/>
+                            <th t-else="" t-att-colspan="cell.width" t-att-rowspan="cell.height" class="border-0 fw-normal bg-100" t-att-class="{ 'border-0': cell_index != 0 }"/>
```

Comment:
Convention: classes order should match CSS declarations one ;)
https://www.odoo.com/documentation/master/developer/howtos/scss_tips.html#handling-utility-classes-verbosity